Mathlib/CategoryTheory/DiscreteCategory.lean

```diff
@@ -167,9 +167,9 @@ theorem functor_obj {I : Type u₁} (F : I → C) (i : I) :
     (Discrete.functor F).obj (Discrete.mk i) = F i :=
   rfl
 
-@[simp]
 theorem functor_map {I : Type u₁} (F : I → C) {i : Discrete I} (f : i ⟶ i) :
     (Discrete.functor F).map f = 𝟙 (F i.as) := by aesop_cat
```

Comment:
So this works, right?
```suggestion
    (Discrete.functor F).map f = 𝟙 (F i.as) := by simp
```

Reply:
Might not work be because `Discrete.functor_map_id` is below this lemma.